integration_tests/integration/analytics/protectedCharacteristics.spec.ts

```diff
@@ -147,47 +161,37 @@ context('Analytics section > Protected characteristics page', () => {
     const gaSpy = new GoogleAnalyticsSpy()
     gaSpy.install()
 
-    page
-      .getChartGuidance('incentive-levels-by-age')
-      .click()
-      .then(() =>
-        gaSpy.shouldHaveSentEvent('incentives_event', {
-          category: 'How you can use this chart > Incentive level by age',
-          action: 'opened',
-          label: 'MDI',
-        }),
-      )
-    page
-      .getChartGuidance('incentive-levels-by-age')
-      .click()
-      .then(() =>
-        gaSpy.shouldHaveSentEvent('incentives_event', {
-          category: 'How you can use this chart > Incentive level by age',
-          action: 'closed',
-          label: 'MDI',
-        }),
-      )
-
-    page
-      .getChartGuidance('prisoners-with-entries-by-age')
-      .click()
-      .then(() =>
-        gaSpy.shouldHaveSentEvent('incentives_event', {
-          category: 'How you can use this chart > Behaviour entries by age',
-          action: 'opened',
-          label: 'MDI',
-        }),
-      )
-    page
-      .getChartGuidance('prisoners-with-entries-by-age')
-      .click()
-      .then(() =>
-        gaSpy.shouldHaveSentEvent('incentives_event', {
-          category: 'How you can use this chart > Behaviour entries by age',
-          action: 'closed',
-          label: 'MDI',
-        }),
-      )
+    const guidanceBoxes: [ChartId, string][] = [
+      ['incentive-levels-by-age', 'How you can use this chart > Incentive level by age'],
```

Comment:
Thought to do a similar test style to `feedback box for analytics is tracked`

Worth noting that we don't have a guidance box for `population-by-age` hence testing the other 5 only